### ISSUE TITLE
Issue with Wallaby with webpack with Jasmine Tests - "Cannot access before initialization"

### ISSUE THREAD

**Issue by eonarheim** [wallabyjs/public#3314]
### Issue description or question

Hello! Big fan of wallaby, loyal customer and advocate for many years! I have an issue happening on a feature branch of my repo where it seems to complain about init order 

```sh
Runtime error: ReferenceError: .\src\engine\Particles.ts : Cannot access 'Actor' before initialization​​
```
<img width="601" alt="image" src="https://github.com/wallabyjs/public/assets/612071/9f513b81-b8e3-4111-a487-302053e73015">

However my normal webpack builds/tests run and produce artifacts just fine without the same issue, so it seems to be isolated to wallaby?
* `npm run start`
* `npm run build:esm`
* `npm run test`

Here is the PR https://github.com/excaliburjs/Excalibur/pull/2790/files. The issue can be reproduced on that branch/commit by starting wallaby in VSCode

Commit [8e2bc9ea2c555dbbd0cb7431825fbf1f79dd0b00 ](https://github.com/excaliburjs/Excalibur/pull/2790/commits/8e2bc9ea2c555dbbd0cb7431825fbf1f79dd0b00)is the latest with the issue.

I've tried refactoring the imports to be exact instead of barrel imports but that doesn't seem to help.

### Wallaby diagnostics report

```
{
  editorVersion: '1.85.1',
  pluginVersion: '1.0.371',
  editorType: 'VSCode',
  osVersion: 'win32 10.0.22621',
  nodeVersion: 'v18.14.1',
  coreVersion: '1.0.1517',
  checksum: 'MjJlZTY2NThhMTA1NzNhNjQzOTVjOTdlNWIwYmMyZGYsMTczMjQ5MjgwMDAwMCww',
  config: {
    files: [
      { pattern: 'src/spec/util/*.ts', load: false, ignore: false, trigger: true, order: 1 },
      { pattern: 'src/engine/**/*.ts', load: false, ignore: false, trigger: true, order: 2 },
      { pattern: 'src/engine/**/*.glsl', load: false, ignore: false, trigger: true, order: 3 },
      { pattern: 'src/spec/images/**/*.mp3', ignore: false, trigger: true, load: true, order: 4 },
      { pattern: 'src/spec/images/**/*.ogg', ignore: false, trigger: true, load: true, order: 5 },
      { pattern: 'src/spec/images/**/*.png', ignore: false, trigger: true, load: true, order: 6 },
      { pattern: 'src/spec/images/**/*.gif', ignore: false, trigger: true, load: true, order: 7 },
      { pattern: 'src/spec/images/**/*.txt', ignore: false, trigger: true, load: true, order: 8 },
      { pattern: 'src/spec/images/**/*.css', ignore: false, trigger: true, load: true, order: 9 },
      { pattern: 'src/spec/images/**/*.woff2', ignore: false, trigger: true, load: true, order: 10 }
    ],
    tests: [ { pattern: 'src/spec/*Spec.ts', ignore: false, trigger: true, load: true, test: true, order: 11 } ],
    env: {
      kind: 'chrome',
      runner: './node_modules/puppeteer/.local-chromium/win64-1022525/chrome-win/chrome.exe',
      params: { runner: '--headless --mute-audio --autoplay-policy=no-user-gesture-required' },
      type: 'browser',
      viewportSize: { width: 800, height: 600 },
      options: { width: 800, height: 600 },
      bundle: true
    },
    testFramework: { version: 'jasmine@3.10.0', configurator: 'jasmine@2.1.3', reporter: 'jasmine@2.1.3', starter: 'jasmine@2.1.3' },
    diagnostics: {},
    captureConsoleLog: true,
    filesWithNoCoverageCalculated: [],
    runAllTestsInAffectedTestFile: false,
    updateNoMoreThanOneSnapshotPerTestFileRun: false,
    compilers: {},
    logLimits: { inline: { depth: 5, elements: 5000 }, values: { default: { stringLength: 200 }, autoExpand: { elements: 5000, stringLength: 8192, depth: 10 } } },
    preprocessors: {},
    maxConsoleMessagesPerTest: 100,
    autoConsoleLog: true,
    delays: { run: 0, edit: 100, update: 0 },
    workers: { initial: 0, regular: 0, recycle: false },
    teardown: undefined,
    hints: {
      ignoreCoverage: '__REGEXP /ignore coverage|istanbul ignore|c8 ignore/',
      ignoreCoverageForFile: '__REGEXP /ignore file coverage/',
      commentAutoLog: '?',
      testFileSelection: { include: '__REGEXP /file\\.only/', exclude: '__REGEXP /file\\.skip/' }
    },
    automaticTestFileSelection: true,
    runSelectedTestsOnly: false,
    mapConsoleMessagesStackTrace: false,
    extensions: {},
    reportUnhandledPromises: false,
    throwOnBeforeUnload: true,
    slowTestThreshold: 75,
    lowCoverageThreshold: 80,
    runAllTestsWhenNoAffectedTests: false,
    configCode: "const path = require('path');\r\n" +
      "const webpack = require('webpack');\r\n" +
      '\r\n' +
      'module.exports = function (wallaby) {\r\n' +
      '  return {\r\n' +
      '    files: [\r\n' +
      "      { pattern: 'src/spec/util/*.ts', load: false },\r\n" +
      "      { pattern: 'src/engine/**/*.ts', load: false },\r\n" +
      "      { pattern: 'src/engine/**/*.glsl', load: false },\r\n" +
      "      { pattern: 'src/spec/images/**/*.mp3' },\r\n" +
      "      { pattern: 'src/spec/images/**/*.ogg' },\r\n" +
      "      { pattern: 'src/spec/images/**/*.png' },\r\n" +
      "      { pattern: 'src/spec/images/**/*.gif' },\r\n" +
      "      { pattern: 'src/spec/images/**/*.txt' },\r\n" +
      "      { pattern: 'src/spec/images/**/*.css' },\r\n" +
      "      { pattern: 'src/spec/images/**/*.woff2' }\r\n" +
      '    ],\r\n' +
      "    tests: ['./src/spec/*Spec.ts'],\r\n" +
      '    env: {\r\n' +
      "      kind: 'chrome',\r\n" +
      '      // This is tied to the puppeteer install\r\n' +
      "      runner: './node_modules/puppeteer/.local-chromium/win64-1022525/chrome-win/chrome.exe',\r\n" +
      '      params: {\r\n' +
      "        runner: '--headless --mute-audio --autoplay-policy=no-user-gesture-required'\r\n" +
      '      }\r\n' +
      '    },\r\n' +
      "    testFramework: 'jasmine',\r\n" +
      '    setup: function () {\r\n' +
      '      window.__moduleBundler.loadTests();\r\n' +
      '    },\r\n' +
      '    postprocessor: wallaby.postprocessors.webpack({\r\n' +
      "      mode: 'none',\r\n" +
      "      devtool: 'source-map',\r\n" +
      '      resolve: {\r\n' +
      "        extensions: ['.ts', '.js'],\r\n" +
      '        alias: {\r\n' +
      "          '@excalibur': path.resolve(__dirname, './src/engine/')\r\n" +
      '        }\r\n' +
      '      },\r\n' +
      '      plugins: [\r\n' +
      '        new webpack.DefinePlugin({\r\n' +
      `          'process.env.__EX_VERSION': "'test-runner'"\r\n` +
      '        })\r\n' +
      '      ],\r\n' +
      '      module: {\r\n' +
      '        rules: [\r\n' +
      '          {\r\n' +
      '            test: /\\.ts$/,\r\n' +
      "            loader: 'ts-loader',\r\n" +
      '            options: {\r\n' +
      '              transpileOnly: true,\r\n' +
      '              projectReferences: true,\r\n' +
      "              configFile: 'tsconfig.json'\r\n" +
      '            }\r\n' +
      '          },\r\n' +
      '          {\r\n' +
      '            test: /\\.css$/,\r\n' +
      "            use: ['css-loader']\r\n" +
      '          },\r\n' +
      '          {\r\n' +
      '            test: /\\.(png|jpg|gif|mp3)$/i,\r\n' +
      '            use: [\r\n' +
      '              {\r\n' +
      "                loader: 'url-loader',\r\n" +
      '                options: {\r\n' +
      '                  limit: 8192\r\n' +
      '                }\r\n' +
      '              }\r\n' +
      '            ]\r\n' +
      '          },\r\n' +
      '          {\r\n' +
      '            test: /\\.(glsl)$/,\r\n' +
      "            use: ['raw-loader']\r\n" +
      '          }\r\n' +
      '        ]\r\n' +
      '      }\r\n' +
      '    })\r\n' +
      '  };\r\n' +
      '};\r\n'
  },
  packageJSON: {
    dependencies: { 'core-js': '3.33.3' },
    devDependencies: {
      '@babel/core': '7.23.7',
      '@fortawesome/fontawesome-free': '6.5.1',
      '@octokit/rest': '19.0.13',
      '@storybook/addon-actions': '6.5.16',
      '@storybook/addon-essentials': '6.5.16',
      '@storybook/addon-links': '6.5.16',
      '@storybook/builder-webpack5': '6.5.16',
      '@storybook/html': '6.5.16',
      '@storybook/manager-webpack5': '6.5.16',
      '@types/jasmine': '5.1.4',
      '@types/node': '18.19.4',
      '@types/react-color': '3.0.10',
      '@types/webpack-env': '1.18.4',
      '@typescript-eslint/eslint-plugin': '6.13.1',
      '@typescript-eslint/parser': '6.13.1',
      'babel-loader': '9.1.3',
      'copy-webpack-plugin': '11.0.0',
      copyfiles: '2.4.1',
      coveralls: '3.1.1',
      'cross-env': '7.0.3',
      'css-loader': '6.8.1',
      eslint: '8.55.0',
      'eslint-config-prettier': '9.0.0',
      'eslint-plugin-jsdoc': '46.9.0',
      'excalibur-jasmine': '0.2.0',
      istanbul: '0.4.5',
      'istanbul-instrumenter-loader': '3.0.1',
      jasmine: '5.1.0',
      'jasmine-core': '5.1.1',
      karma: '6.4.2',
      'karma-browserstack-launcher': '1.6.0',
      'karma-chrome-launcher': '3.2.0',
      'karma-coverage': '2.2.1',
      'karma-coverage-istanbul-reporter': '3.0.3',
      'karma-jasmine': '5.1.0',
      'karma-summary-reporter': '3.1.1',
      'karma-webpack': '5.0.0',
      puppeteer: '15.5.0',
      react: '17.0.2',
      'react-dom': '17.0.2',
      'replace-in-file': '7.0.2',
      rimraf: '5.0.5',
      semver: '7.5.4',
      serve: '14.2.1',
      'source-map-support': '0.5.21',
      'sync-request': '6.1.0',
      'terser-webpack-plugin': '5.3.10',
      'ts-loader': '9.5.1',
      typedoc: '0.25.3',
      typescript: '5.3.3',
      'url-loader': '4.1.1',
      'wallaby-webpack': '3.9.16',
      webpack: '5.89.0',
      'webpack-cli': '5.1.4'
    }
  },
  fs: { numberOfFiles: 567 },
  debug: [
    '2024-01-08T16:01:17.301Z project waiting for initial run signal\n',
    '2024-01-08T16:01:17.359Z project Wallaby Node version: v18.14.1\n',
    '2024-01-08T16:01:17.359Z project Wallaby config: C:\\projects\\excalibur-mainline\\wallaby.js\n',
    '2024-01-08T16:01:17.432Z fs File system starting\n',
    '2024-01-08T16:01:18.072Z fs File system scan completed\n',
    '2024-01-08T16:01:18.079Z project File cache: C:\\Users\\eriko\\.vscode\\extensions\\wallabyjs.wallaby-vscode-1.0.371\\projects\\7aaad031a933a5cd\n',
    '2024-01-08T16:01:18.249Z uiService Listening port 51235\n',
    '2024-01-08T16:01:18.375Z chromeRunner launch new instance (path: ./node_modules/puppeteer/.local-chromium/win64-1022525/chrome-win/chrome.exe, flags: --headless --mute-audio --autoplay-policy=no-user-gesture-required)\n',
    '2024-01-08T16:01:18.378Z workers Parallelism for initial run: 6, for regular run: 3\n',
    '2024-01-08T16:01:18.379Z workers Starting run worker instance #0\n',
    '2024-01-08T16:01:18.379Z workers Starting run worker instance #1\n',
    '2024-01-08T16:01:18.379Z workers Starting run worker instance #2\n',
    '2024-01-08T16:01:18.379Z workers Starting run worker instance #3\n',
    '2024-01-08T16:01:18.379Z workers Starting run worker instance #4\n',
    '2024-01-08T16:01:18.379Z workers Starting run worker instance #5\n',
    '2024-01-08T16:01:18.381Z workers Web server is listening at 64419\n',
    '2024-01-08T16:01:18.401Z project Stopping process pool\n',
    '2024-01-08T16:01:18.401Z project File cache is up-to-date, starting full test run\n',
    '2024-01-08T16:01:18.402Z chromeRunner Connection to Chrome Debugger Protocol on 127.0.0.1:64418 failed: Error: connect ECONNREFUSED 127.0.0.1:64418\n',
    '2024-01-08T16:01:18.404Z project Running postprocessor\n',
    '2024-01-08T16:01:18.414Z postprocessor Processing started.\n',
    '2024-01-08T16:01:18.415Z postprocessor Initializing.\n',
    '2024-01-08T16:01:18.416Z postprocessor Initializing entry.\n',
    '2024-01-08T16:01:18.416Z postprocessor Sanitizing configuration.\n',
    '2024-01-08T16:01:18.417Z postprocessor Adjusting configuration.\n',
    '2024-01-08T16:01:18.417Z postprocessor Creating compiler.\n',
    '2024-01-08T16:01:18.980Z postprocessor Customizing compiler.\n',
    '2024-01-08T16:01:18.980Z postprocessor Compilation started.\n',
    '2024-01-08T16:01:19.007Z chromeRunner Connection to Chrome Debugger Protocol on 127.0.0.1:64418 succeeded\n',
    '2024-01-08T16:01:19.007Z workers Started run worker instance (immediate) #0\n',
    '2024-01-08T16:01:19.007Z workers Started run worker instance (immediate) #1\n',
    '2024-01-08T16:01:19.007Z workers Started run worker instance (immediate) #2\n',
    '2024-01-08T16:01:19.007Z workers Started run worker instance (immediate) #3\n',
    '2024-01-08T16:01:19.007Z workers Started run worker instance (immediate) #4\n',
    '2024-01-08T16:01:19.007Z workers Started run worker instance (immediate) #5\n',
    '2024-01-08T16:01:19.013Z fs File changed in editor: src/engine/Graphics/PostProcessor/PostProcessor.ts\n',
    '2024-01-08T16:01:33.120Z postprocessor Compilation finished.\n',
    '2024-01-08T16:01:34.507Z postprocessor Emitting 432 files.\n',
    '2024-01-08T16:01:34.565Z postprocessor Processing finished.\n',
    '2024-01-08T16:01:34.584Z project Postprocessor execution finished\n',
    '2024-01-08T16:01:34.584Z project Test run started; run priority: 3\n',
    '2024-01-08T16:01:34.591Z project Running all tests\n',
    '2024-01-08T16:01:34.603Z workers Starting test run, priority: 3\n',
    '2024-01-08T16:01:34.603Z workers Distributing tests between 6 workers\n',
    '2024-01-08T16:01:34.604Z workers Running tests in parallel\n',
    '2024-01-08T16:01:34.604Z chromeRunner Starting sandbox [worker #0, session #xgwgc]\n',
    '2024-01-08T16:01:34.604Z chromeRunner Starting sandbox [worker #1, session #ncxr5]\n',
    '2024-01-08T16:01:34.604Z chromeRunner Starting sandbox [worker #2, session #mpskk]\n',
    '2024-01-08T16:01:34.604Z chromeRunner Starting sandbox [worker #3, session #sneql]\n',
    '2024-01-08T16:01:34.604Z chromeRunner Starting sandbox [worker #4, session #enngv]\n',
    '2024-01-08T16:01:34.604Z chromeRunner Starting sandbox [worker #5, session #up1qk]\n',
    '2024-01-08T16:01:34.604Z chromeRunner Preparing sandbox [worker #0, session #xgwgc]\n',
    '2024-01-08T16:01:34.606Z browserRunner Total files to load in sandbox: 230\n',
    '2024-01-08T16:01:34.607Z browserRunner Sandbox is generated [worker #0, session #xgwgc]: http://localhost:64419/wallaby_sandbox0.html\n',
    '2024-01-08T16:01:34.607Z chromeRunner Preparing sandbox [worker #1, session #ncxr5]\n',
    '2024-01-08T16:01:34.607Z browserRunner Total files to load in sandbox: 230\n',
    '2024-01-08T16:01:34.608Z browserRunner Sandbox is generated [worker #1, session #ncxr5]: http://localhost:64419/wallaby_sandbox1.html\n',
    '2024-01-08T16:01:34.608Z chromeRunner Preparing sandbox [worker #2, session #mpskk]\n',
    '2024-01-08T16:01:34.609Z browserRunner Total files to load in sandbox: 230\n',
    '2024-01-08T16:01:34.610Z browserRunner Sandbox is generated [worker #2, session #mpskk]: http://localhost:64419/wallaby_sandbox2.html\n',
    '2024-01-08T16:01:34.610Z chromeRunner Preparing sandbox [worker #3, session #sneql]\n',
    '2024-01-08T16:01:34.610Z browserRunner Total files to load in sandbox: 230\n',
    '2024-01-08T16:01:34.611Z browserRunner Sandbox is generated [worker #3, session #sneql]: http://localhost:64419/wallaby_sandbox3.html\n',
    '2024-01-08T16:01:34.611Z chromeRunner Preparing sandbox [worker #4, session #enngv]\n',
    '2024-01-08T16:01:34.612Z browserRunner Total files to load in sandbox: 231\n',
    '2024-01-08T16:01:34.613Z browserRunner Sandbox is generated [worker #4, session #enngv]: http://localhost:64419/wallaby_sandbox4.html\n',
    '2024-01-08T16:01:34.613Z chromeRunner Preparing sandbox [worker #5, session #up1qk]\n',
    '2024-01-08T16:01:34.613Z browserRunner Total files to load in sandbox: 230\n',
    '2024-01-08T16:01:34.614Z browserRunner Sandbox is generated [worker #5, session #up1qk]: http://localhost:64419/wallaby_sandbox5.html\n',
    '2024-01-08T16:01:34.614Z chromeRunner Prepared sandbox [worker #0, session #xgwgc]\n',
    '2024-01-08T16:01:34.614Z chromeRunner Prepared sandbox [worker #1, session #ncxr5]\n',
    '2024-01-08T16:01:34.614Z chromeRunner Prepared sandbox [worker #2, session #mpskk]\n',
    '2024-01-08T16:01:34.614Z chromeRunner Prepared sandbox [worker #3, session #sneql]\n',
    '2024-01-08T16:01:34.614Z chromeRunner Prepared sandbox [worker #4, session #enngv]\n',
    '2024-01-08T16:01:34.614Z chromeRunner Prepared sandbox [worker #5, session #up1qk]\n',
    '2024-01-08T16:01:34.614Z workers [worker #0, session #xgwgc] Running tests in sandbox\n',
    '2024-01-08T16:01:34.614Z workers [worker #1, session #ncxr5] Running tests in sandbox\n',
    '2024-01-08T16:01:34.615Z workers [worker #2, session #mpskk] Running tests in sandbox\n',
    '2024-01-08T16:01:34.615Z workers [worker #3, session #sneql] Running tests in sandbox\n',
    '2024-01-08T16:01:34.615Z workers [worker #4, session #enngv] Running tests in sandbox\n',
    '2024-01-08T16:01:34.615Z workers [worker #5, session #up1qk] Running tests in sandbox\n',
    "2024-01-08T16:01:35.451Z workers Sandbox (active) [xgwgc] error: ReferenceError: .\\src\\engine\\Particles.ts : Cannot access 'Actor' before initialization\n" +
      '    at Module.Actor (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:3203:52)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:5291:68\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:5452:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:11768:68\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:11975:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:777:73\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:931:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:12617:59\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:12664:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:3220:75\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:3699:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6051:64\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6139:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:5460:73\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:5849:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:20562:64\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:20654:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6740:72\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6768:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:1214:80\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:1397:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:15104:77\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:15396:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6822:72\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6973:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:2534:74\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:3200:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:1711:65\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:2524:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:464:65\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:711:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/src/spec/ActionSpec.ts.wbp.js?1704729693147&wallabyFileId=1:2:68\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/src/spec/ActionSpec.ts.wbp.js?1704729693147&wallabyFileId=1:1205:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-webpack-bootstrapper.js?1704729694507:19:36\n' +
      '    at http://localhost:64419/__wallaby__/tracer.js:14:49008\n' +
      '    at l._doWhenReceiverIsReady (http://localhost:64419/__wallaby__/tracer.js:14:12412)\n' +
      '    at Object._startWhenReceiverIsReady (http://localhost:64419/__wallaby__/tracer.js:14:48974)\n' +
      '    at window.__moduleBundler.loadTests (http://localhost:64419/____wallaby-webpack-bootstrapper.js?1704729694507:14:22)\n' +
      '    at http://localhost:64419/wallaby_sandbox0.html:10:30\n' +
      '    at http://localhost:64419/wallaby_sandbox0.html:11:7\n',
    '2024-01-08T16:01:35.455Z workers [xgwgc] Loaded 0 test(s)\n',
    "2024-01-08T16:01:35.462Z workers Sandbox (active) [ncxr5] error: ReferenceError: .\\src\\engine\\Particles.ts : Cannot access 'Actor' before initialization\n" +
      '    at Module.Actor (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:3203:52)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:5291:68\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:5452:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:11768:68\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:11975:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:777:73\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:931:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:12617:59\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:12664:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:3220:75\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:3699:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6051:64\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6139:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:5460:73\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:5849:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:20562:64\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:20654:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6740:72\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6768:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:1214:80\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:1397:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:15104:77\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:15396:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6822:72\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6973:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:2534:74\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:3200:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:1711:65\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:2524:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:464:65\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:711:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/src/spec/ActorSpec.ts.wbp.js?1704729693212&wallabyFileId=2:2:68\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/src/spec/ActorSpec.ts.wbp.js?1704729693212&wallabyFileId=2:1432:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-webpack-bootstrapper.js?1704729694507:19:36\n' +
      '    at http://localhost:64419/__wallaby__/tracer.js:14:49008\n' +
      '    at l._doWhenReceiverIsReady (http://localhost:64419/__wallaby__/tracer.js:14:12412)\n' +
      '    at Object._startWhenReceiverIsReady (http://localhost:64419/__wallaby__/tracer.js:14:48974)\n' +
      '    at window.__moduleBundler.loadTests (http://localhost:64419/____wallaby-webpack-bootstrapper.js?1704729694507:14:22)\n' +
      '    at http://localhost:64419/wallaby_sandbox1.html:10:30\n' +
      '    at http://localhost:64419/wallaby_sandbox1.html:11:7\n',
    '2024-01-08T16:01:35.464Z workers [ncxr5] Loaded 0 test(s)\n',
    "2024-01-08T16:01:35.467Z workers Sandbox (active) [mpskk] error: ReferenceError: .\\src\\engine\\Particles.ts : Cannot access 'Actor' before initialization\n" +
      '    at Module.Actor (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:3203:52)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:5291:68\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:5452:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:11768:68\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:11975:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:777:73\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:931:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:12617:59\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:12664:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:3220:75\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:3699:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6051:64\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6139:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:5460:73\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:5849:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:20562:64\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:20654:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6740:72\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6768:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:1214:80\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:1397:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:15104:77\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:15396:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6822:72\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6973:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:2534:74\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:3200:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:1711:65\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:2524:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:464:65\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:711:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/src/spec/AffineMatrixSpec.ts.wbp.js?1704729693269&wallabyFileId=3:2:68\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/src/spec/AffineMatrixSpec.ts.wbp.js?1704729693269&wallabyFileId=3:195:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-webpack-bootstrapper.js?1704729694507:19:36\n' +
      '    at http://localhost:64419/__wallaby__/tracer.js:14:49008\n' +
      '    at l._doWhenReceiverIsReady (http://localhost:64419/__wallaby__/tracer.js:14:12412)\n' +
      '    at Object._startWhenReceiverIsReady (http://localhost:64419/__wallaby__/tracer.js:14:48974)\n' +
      '    at window.__moduleBundler.loadTests (http://localhost:64419/____wallaby-webpack-bootstrapper.js?1704729694507:14:22)\n' +
      '    at http://localhost:64419/wallaby_sandbox2.html:10:30\n' +
      '    at http://localhost:64419/wallaby_sandbox2.html:11:7\n',
    '2024-01-08T16:01:35.468Z workers [mpskk] Loaded 0 test(s)\n',
    "2024-01-08T16:01:35.475Z workers Sandbox (active) [enngv] error: ReferenceError: .\\src\\engine\\Particles.ts : Cannot access 'Actor' before initialization\n" +
      '    at Module.Actor (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:3203:52)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:5291:68\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:5452:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:11768:68\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:11975:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:777:73\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:931:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:12617:59\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:12664:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:3220:75\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:3699:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6051:64\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6139:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:5460:73\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:5849:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:20562:64\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:20654:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6740:72\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6768:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:1214:80\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:1397:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:15104:77\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:15396:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6822:72\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:6973:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:2534:74\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:3200:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:1711:65\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:2524:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:464:65\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/____wallaby-bundle.js?1704729694573&wallabyFileId=bundle:711:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/src/spec/AnimationSpec.ts.wbp.js?1704729693292&wallabyFileId=5:2:68\n' +
      '    at window.__moduleBundler.modules.<computed> (http://localhost:64419/src/spec/AnimationSpec.ts.wbp.js?1704729693292&wallabyFileId=5:592:3)\n' +
      '    at window.__moduleBundler.require (http://localhost:64419/____wallaby-webpack.js?1704729694507:28:18)\n' +
      '    at http://localhost:64419/____wallaby-webpack-bootstrapper.js?1704729694507:19:36\n' +
      '    at http://localhost:64419/__wallaby__/tracer.js:14:49008\n' +
      '    at l._doWhenReceiverIsReady (http://localhost:64419/__wallaby__/tracer.js:14:12412)\n' +
      '    at Object._startWhenReceiverIsReady (http://localhost:64419/__wallaby__/tracer.js:14:48974)\n' +
      '    at window.__moduleBundler.loadTests (http://localhost:64419/____wallaby-webpack-bootstrapper.js?1704729694507:14:22)\n' +
      '    at http://localhost:64419/wallaby_sandbox4.html:10:30\n' +
      '    at http://localhost:64419/wallaby_sandbox4.html:11:7\n',
    '2024-01-08T16:03:10.800Z workers [hrulo] Run 0 test(s), skipped 0 test(s)\n',
    '2024-01-08T16:03:10.814Z workers [hrulo] Sandbox is responsive, closing it\n',
    '2024-01-08T16:03:10.814Z chromeRunner Closing tab as requested\n',
    '2024-01-08T16:03:10.814Z chromeRunner Closing Chrome Tab\n',
    '2024-01-08T16:03:10.847Z workers [0yu01] Loaded 0 test(s)\n',
    '2024-01-08T16:03:10.853Z workers [9yfca] Run 0 test(s), skipped 0 test(s)\n',
    '2024-01-08T16:03:10.864Z workers [9yfca] Sandbox is responsive, closing it\n',
    '2024-01-08T16:03:10.864Z chromeRunner Closing tab as requested\n',
    '2024-01-08T16:03:10.864Z chromeRunner Closing Chrome Tab\n',
    '2024-01-08T16:03:10.892Z workers [0yu01] Run 0 test(s), skipped 0 test(s)\n',
    '2024-01-08T16:03:10.980Z workers [0yu01] Sandbox is responsive, closing it\n',
    '2024-01-08T16:03:10.980Z chromeRunner Closing tab as requested\n',
    '2024-01-08T16:03:10.980Z chromeRunner Closing Chrome Tab\n',
    '2024-01-08T16:03:10.980Z workers Merging parallel test run results\n',
    '2024-01-08T16:03:10.986Z project Test run finished\n',
    '2024-01-08T16:03:10.986Z project Processed console.log entries\n',
    '2024-01-08T16:03:10.986Z project Processed loading sequences\n',
    '2024-01-08T16:03:10.986Z project Processed executed tests\n',
    '2024-01-08T16:03:11.009Z project Processed code coverage\n',
    '2024-01-08T16:03:11.019Z project Test run result processed and sent to IDE\n'
  ]
}
```

^ Bumped into the GH max char limit so truncated some of the log

**Comment by eonarheim** [comment]
Specifically commenting out the import and the 2 lines of the code block that use it seem to make wallaby function again?

https://github.com/excaliburjs/Excalibur/pull/2790/commits/8e2bc9ea2c555dbbd0cb7431825fbf1f79dd0b00#diff-ace4e776131cb973b0f1154d3d491a8c69316146e81955a3f154b13aa4e4fc0a

**Comment by eonarheim** [comment]
@NikGovorov Thanks! That appears to have done the trick!

Thanks for the help! And the `wallaby-webpack` tip 🎉

**Comment by eonarheim** [comment]
Off topic and I can make a new issue if you like!

If you have any perf tips for me let me know (my tests are bonkers and do image diffing/snapshots).

Is there a way I can display image diffs in the wallaby app? Will gladly write up a feature request 😄
